TechnicalDocuments/NewHiresCheckList.md

```diff
@@ -54,15 +52,17 @@ Here's how to get the iOS project up and running.
      brew install git-lfs
      git lfs install
      ```
+1. [Generate](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key) & add the SSH key to your [GitHub account](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account)
 
-1. Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios
+1. Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios  
+*Make sure you've already done the `Setup Git LFS` step before cloning the project.*
      ```
      git clone git@github.com:Babylonpartners/babylon-ios.git
      ```
 
 1. Install RVM and Ruby version 2.4
      ```
-     \curl -sSL https://get.rvm.io | bash -s stable
+     curl -sSL https://get.rvm.io | bash -s stable
```

Comment:
Why did you remove the leading `\` ? This is means it will really run `curl` and ignore any alias you might have.

Reply:
I didn't know that as I thought it's there by mistake. I believe @AliSoftware also thought so :D 
bringing it back

Reply:
Oh, TIL indeed.

Reply:
It was also TIL for me when I first saw this on the RVM site 😄

Reply:
@TheAdamBorek If we put it back, I thonk it would then be worth mentioning the trick using a `# The backslash ensures this command will run the real curl and not an alias` or something? as many would otherwise assume it is a typo